guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -12,38 +12,65 @@ To create a file type repository in a directory on a remote server, see xref:Cre
 
 To create a file type repository in a local directory, complete the following procedure:
 
-. Ensure the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+. Ensure the Server and {project-client-name} repositories are enabled.
```

Comment:
The instructions only speak of the utils, not server and client. Could you make the instructions and the description match?